packages/notification-center/src/store/novu-theme.context.ts

```diff
@@ -67,7 +66,7 @@ export interface IThemeFooter {
   logoPrefixFontColor?: string;
 }
 
-export interface IThemeUnseenBadge {
+export interface INotificationBellTheme {
   color?: {
```

Comment:
@djabarovgeorge maybe we can also just remove the "color" property and put the props in a more flattened way with: borderColor and unseenBadgeBackgroundColor or something like this?

Reply:
Sorry did not saw this one let me look into it :)

Reply:
Oh sorry already did it, I thought you talked about another interface.